src/kvstore/NebulaStore.cpp

```diff
@@ -546,9 +546,9 @@ ResultCode NebulaStore::ingest(GraphSpaceID spaceId) {
             auto files = nebula::fs::FileUtils::listAllFilesInDir(path.c_str(), true, "*.sst");
             for (auto file : files) {
                 LOG(INFO) << "Ingesting extra file: " << file;
-                auto code = engine->ingest(std::vector<std::string>({file}));
-                if (code != ResultCode::SUCCEEDED) {
-                    return code;
+                auto ret_code = engine->ingest(std::vector<std::string>({file}));
+                if (ResultCode::SUCCEEDED != ret_code) {
+                    return ret_code;
```

Comment:
Please don't do changing like this unless it _**really**_ cares.

BTW. Please look around to infer our naming conventions.

Reply:
ok, I will fix it